### PR TITLE
net-mgmt/telegraf: add validation message to tags

### LINKED ISSUE #785: net-mgmt/telegraf - global tag support
OPNsense version : 18.7
os-telegraf plugin : 1.4.0 (package telegraf 1.6.3)

The current OPNsense telegraf service panels don't allow to specify Telegraf global tags.  
It would be helpful to have an entry in the Telegraf General panel, allowing this sort of example in the final telegraf.conf file :
```
[global_tags]
  dc = "UK-LON"
  country = "UK"
  city = "LON"
  virtual_type = "vmware"
  virtual_role = "guest"
  os_family = "freebsd"
  os_distrib = "opnsense"
```
All the tags will be attached to each measurements returned by telegraf, allowing a more precise and generic filtering.

Tags use the following syntax under the `[global_tags]` section :   tag_name = "tag_value"
Order is irrelevant.  
Afaik, both are always handled as string.  
Spaces are also supported, but not recommended (escaping might be required).

### PR DESCRIPTION
**Pull request by mimugmail**
Closes #785

### CODE REVIEW

File: net-mgmt/telegraf/src/opnsense/mvc/app/models/OPNsense/Telegraf/Key.xml [hunk 1]
```diff
@@ -13,11 +13,13 @@
                     <default></default>
                     <Required>N</Required>
                     <mask>/^([0-9a-zA-Z._\-]){1,128}$/u</mask>
+                    <ValidationMessage>Only characters, numbers, a dot, underscore and hyphen allowed. Do not use more than 128 signs.</ValidationMessage>
```
Comment (fichtner) [inline net-mgmt/telegraf/src/opnsense/mvc/app/models/OPNsense/Telegraf/Key.xml]:
```suggestion
                    <ValidationMessage>Only characters, numbers, a dot, underscore and hyphen allowed. Do not use more than 128 characters.</ValidationMessage>
```

File: net-mgmt/telegraf/src/opnsense/mvc/app/models/OPNsense/Telegraf/Key.xml [hunk 2]
```diff
@@ -13,11 +13,13 @@
...
                 </name>
                 <value type="TextField">
                     <default></default>
                     <Required>N</Required>
                     <mask>/^([0-9a-zA-Z._\-]){1,128}$/u</mask>
+                    <ValidationMessage>Only characters, numbers, a dot, underscore and hyphen allowed. Do not use more than 128 signs.</ValidationMessage>
```
Comment (fichtner) [inline net-mgmt/telegraf/src/opnsense/mvc/app/models/OPNsense/Telegraf/Key.xml]:
```suggestion
                    <ValidationMessage>Only characters, numbers, a dot, underscore and hyphen allowed. Do not use more than 128 characters.</ValidationMessage>
```

**Comment by mimugmail** [comment]
Thx for your time guys .. ready for merge. 👍

**Comment by fichtner** [comment]
merged, thanks!

**Pull request merged**